hack/local-up-karmada.sh

```diff
@@ -131,7 +131,11 @@ util::create_cluster "${PULL_MODE_CLUSTER_NAME}" "${PULL_MODE_CLUSTER_TMP_CONFIG
 #step2. make images and get karmadactl
 export VERSION="latest"
 export REGISTRY="docker.io/karmada"
+export KARMADA_IMAGE_LABEL_VALUE="May_be_pruned_in_local-up-karmada.sh"
+export DOCKER_BUILD_ARGS="--label=image.karmada.io=${KARMADA_IMAGE_LABEL_VALUE}"
```

Comment:
Got, no big deal.
Would you like to send a PR for this?

Reply:
> This script should be:
> 
> ```shell
> export DOCKER_BUILD_ARGS="${DOCKER_BUILD_ARGS} --label=image.karmada.io=${KARMADA_IMAGE_LABEL_VALUE}"
> ```

It seems we can not call the `${DOCKER_BUILD_ARGS}` here, since it has never been defined before.

Reply:
Fixed in #4939, PTAL.